### ISSUE TITLE
Change color of the overscroll area and the scrollbar thumb?

### ISSUE THREAD

**Issue by pro-sumer** [louislam/uptime-kuma#4641]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Using CSS I have been able to modify most elements on the status page to match the color scheme of my website, but unfortunately the overscroll area and scrollbar thumb are still displayed in Uptime Kuma's default green (#5cdd8b?) in Safari (macOS, iOS/iPadOS).

Since I can't use the browser's inspector to inspect these elements, I don't know how to modify them. I don't even know whether this can be done in CSS; it might require meta tags? https://webkit.org/blog/11989/new-webkit-features-in-safari-15

How can I change the color of the overscroll area and scrollbar thumb (in modern versions of Safari browsers)?

### 📝 Error Message(s) or Log

N/A

### 🐻 Uptime-Kuma Version

1.23.11

### 💻 Operating System and Arch

macOS Sonoma 14.4.1

### 🌐 Browser

Safari 17.4.1

### 🖥️ Deployment Environment

- Runtime: ? (deployed on Fly.io)
- Database: ? (deployed on Fly.io)
- Filesystem used to store the database on: ? (deployed on Fly.io)
- number of monitors: 28

**Comment by CommanderStorm** [comment]
What are you refering to concretely when you are stating `overscroll area and scrollbar thumb`? Could you attach a screenshot?

**Comment by CommanderStorm** [comment]
I don't quite get what you are refering to when you are saying thumb.

For overscroll there are some areticles out there on how to style this apple-specific thing.
https://camlittle.com/posts/2020-05-15-css-overscroll-styling/

TL;DR you likely want:
```css
html:before {
  content: '';
  position: fixed;
  width: 100%;
  height: 999px;
  top: -999px;
  background-color: tomato;
}
```

**Comment by pro-sumer** [comment]
> I don't quite get what you are refering to when you are saying thumb.

The smaller rectangle in a scrollbar which one can click on and then drag up or down to scroll contents in a page that does not fit the screen.

Does that help in understanding?

(I might be using the wrong term, but I thought it was called "thumb")

**Comment by pro-sumer** [comment]
Hm. Looks like I got it wrong?

Actually the "thumb" is grey and the (remainder of the) scrollbar is green? (opposite of what I thought before)

![Screenshot 2024-04-02 at 14 11 53](https://github.com/louislam/uptime-kuma/assets/44335102/b28db16b-5be2-4a00-9a6b-000bf3857f5d)

**Comment by CommanderStorm** [comment]
You can customise the scrollbar via the [`::-webkit-scrollbar`](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar) and associated nonstandard selectors.

**Comment by pro-sumer** [comment]
Unfortunately both suggested solutions don't seem to work for me.

This was interesting though: https://css-tricks.com/meta-theme-color-and-trickery

I indeed see this in the source code of my status page (which explains the green elements):

```
<meta name="theme-color" id="theme-color" content="#5cdd8b">
```

When I modify the content color in Safari's Web Inspector those bright green elements indeed all get the color I specify.

@CommanderStorm Is there any way I can configure such a meta tag as a user in Uptime Kuma?

**Comment by CommanderStorm** [comment]
You might be able to get around this via the embedded status pages from the wiki, but there are obvious security (more precisely clickjacking) implications of doing so.
